client-js/app/elements/labrad-registry.ts

```diff
@@ -206,34 +207,41 @@ export class LabradRegistry extends polymer.Base {
     if (event.ctrlKey || event.button == 2 ) {
       //should I use switch and case here instead of ifs?
       this.$.dragOp.innerText = 'Copy';
+      this.$.dragDialog.open();
+      window.setTimeout(() => this.$.dragPathInput.$.input.focus(), 0);
```

Comment:
I think you can just use `setTimeout` instead of `window.setTimeout`. We should get in this habit because the former will work no matter what the global object is called, e.g. in unit tests and in the browser, whereas the latter only works in browsers.